packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/configmid.md

```diff
@@ -1,8 +1,10 @@
-Edit `src/main/resources/application.properties` and include your Okta Issuer and Audience from above:
+Configure the Okta Spring Boot Starter by setting the following environment variables:
 
-```properties
-okta.oauth2.issuer=https://${yourOktaDomain}/oauth2/default
+```bash
+OKTA_OAUTH2_ISSUER=${issuer-from-above}
```

Comment:
```suggestion
OKTA_OAUTH2_ISSUER="${issuer-from-above}"
```

Reply:
I changed this so it would match how the other QSes do it. Expect another PR soon